src/Files/Views/Pages/Properties.xaml.cs

```diff
@@ -70,16 +83,84 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
 
         private async void Properties_Loaded(object sender, RoutedEventArgs e)
         {
-            Microsoft.UI.Xaml.Controls.BackdropMaterial.SetApplyToRootOrPageBackground(sender as Control, true);
+            if (!WindowDecorationsHelper.IsWindowDecorationsAllowed) Microsoft.UI.Xaml.Controls.BackdropMaterial.SetApplyToRootOrPageBackground(sender as Control, true);
 
             AppSettings.ThemeModeChanged += AppSettings_ThemeModeChanged;
             if (ApiInformation.IsApiContractPresent("Windows.Foundation.UniversalApiContract", 8))
             {
                 // Set window size in the loaded event to prevent flickering
-                TitleBar = ApplicationView.GetForCurrentView().TitleBar;
-                TitleBar.ButtonBackgroundColor = Colors.Transparent;
-                TitleBar.ButtonInactiveBackgroundColor = Colors.Transparent;
-                Window.Current.SetTitleBar(TitleBarDragArea);
+                if (WindowDecorationsHelper.IsWindowDecorationsAllowed)
+                {
+                    appWindow.TitleBar.SetPreferredVisibility(AppWindowTitleBarVisibility.AlwaysHidden);
+                    appWindow.Frame.DragRegionVisuals.Add(TitleBarDragArea);
+
+                    crossIcon.Foreground = ThemeHelper.RootTheme switch
+                    {
+                        ElementTheme.Default => new SolidColorBrush((Color)Application.Current.Resources["SystemBaseHighColor"]),
+                        ElementTheme.Light => new SolidColorBrush(Colors.Black),
+                        ElementTheme.Dark => new SolidColorBrush(Colors.White),
+                        _ => new SolidColorBrush((Color)Application.Current.Resources["SystemBaseHighColor"])
+                    };
+
+                    AppWindowTitleBarCaptionButtonsGrid.Visibility = Visibility.Visible;
```

Comment:
Can you use conditional xaml in the xaml file instead of this?

Reply:
nope, we can't use custom functions/conditions in conditional XAML
We can bind the property but it doesn't worth it, that will also increase the complexity
If we really need to use binding I would suggest binding x:Load instead
What do you think?